src/components/Button/Button.js

```diff
@@ -1,28 +1,22 @@
-/* eslint-disable react/prefer-stateless-function */
 /* eslint-disable object-curly-newline */
```

Comment:
[OPTIONAL] I think you can easily fix this linter error, remember that linters are here to help us keep the code clean and standardized across teams and disabling rules defeats this purpose.